app/helpers/podcast_metrics_helper.rb

```diff
@@ -0,0 +1,96 @@
+module PodcastMetricsHelper
+  def line_chart_colors
+    ["#007EB2", "#FF9600", "#75BBE1", "#FFC107", "#6F42C1", "#DC3545", "#198754", "#D63384", "#20C997", "#555555"]
+  end
+
+  def parse_agent_data(agents)
+    agents.map do |a|
+      {
+        x: a.label,
+        y: a.count
+      }
+    end
+  end
+
+  def sum_rollups(rollups)
+    rollups.map { |r| r[:count] }.reduce(:+)
+  end
+
+  def interval_options(date_start, date_end)
+    day_count = (date_start.to_date..date_end.to_date).to_a.length
+    if day_count > 60
+      [
+        ["Daily", "DAY"],
+        ["Weekly", "WEEK"],
+        ["Monthly", "MONTH"]
+      ]
+    elsif day_count < 30
+      [
+        ["Hourly", "HOUR"],
+        ["Daily", "DAY"],
+        ["Weekly", "WEEK"]
+      ]
+    else
+      [
+        ["Hourly", "HOUR"],
+        ["Daily", "DAY"],
+        ["Weekly", "WEEK"],
+        ["Monthly", "MONTH"]
+      ]
+    end
+  end
+
+  def rollups_date_range_options
```

Comment:
simple presets to change the start date. like the interval options above, could detail this later, to make sure certain date ranges are only available with certain intervals.